examples/flutter_gallery/lib/demo/material/chip_demo.dart

```diff
@@ -319,7 +318,7 @@ class _ChipDemoState extends State<ChipDemo> {
       body: ChipTheme(
         data: _showShapeBorder
             ? theme.chipTheme.copyWith(
-                shape: BeveledRectangleBorder(
+              shape: BeveledRectangleBorder(
```

Comment:
The original indent should have been left alone: `shape` is a copyWith parameter.